src/main/java/org/levelup/labradoodle/controllers/HomeController.java

```diff
@@ -69,4 +69,22 @@ public DishesDto getDishById(@PathVariable(value = "id") int id) {
         }
         return typeDishes;
     }
+
+    /**
+     * This method return Dish (WEB model) from DB by id. Current version work with mock
+     * @author Barkovskiy Alexandr
+     * @return Dish
+     */
+    @ResponseBody
+    @RequestMapping(value = "/getFirst20",method = RequestMethod.GET)
```

Comment:
Don't name urls with camel case naming pattern.. The count of returning dishes shouldn't be in url\method names... It's just returning dishes, count contains in some constant, so if we change count we wouldn't need rename all of this